bin.src/showPsfs.py

```diff
@@ -329,7 +339,8 @@ def frameInfoFrom(filepath):
             if mat:
                 v1 = int(mat.group(1))
                 v2 = int(mat.group(2))
-                v3 = mat.group(3); v3 = int(v3) if v3 else 1
+                v3 = mat.group(3)
+                v3 = int(v3) if v3 else 1
```

Comment:
Hmm... That could probably go either way. Sadly, it's another untestable `bin/` file that I'd rather not muck around in.